### PR TITLE
DOC: 7168 issue, binom_test example

### PR DESCRIPTION
**Pull request by rafalalgo**
Reference: #7168

### CODE REVIEW

**Comment by chrisb83** [comment]
I think it would be helpful to add a sentence to explain the examples (like "Assume one tosses a fair coin 100 times and ... etc" ).

File: scipy/stats/morestats.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def bayes_mvs(data, alpha=0.90):
     References
     ----------
     T.E. Oliphant, "A Bayesian perspective on estimating mean, variance, and
-    standard-deviation from data", https://scholarsarchive.byu.edu/facpub/278,
+    standard-deviation from data", http://scholarsarchive.byu.edu/facpub/278,
```
Comment (rgommers) [inline scipy/stats/morestats.py]: why are these changed back to `http`?

**Comment by rgommers** [comment]
this is stalled, no action in 6 months. the http link changes are incorrect, and the example doesn't add much (they're just single numbers filled in, it doesn't illustrate much about how `binom_test` works). so closing. thanks for trying to contribute anyway @rafalalgo
